Fix incompatible mitt library version is compatible with vue version

Jzow opened the pull request:
Modifying two files that are incompatible with the `vue` version and result in syntax errors

Jzow commented:
CC #106 

Github CI/CD run error: 
```
Error: Dependencies lock file is not found in XX 
Supported file patterns: package-lock.json,npm-shrinkwrap.json,yarn.lock
```

It is necessary to discuss whether to add a `package-lock` file.

Jzow commented on .github/workflows/node.js.yml:
```diff
@@ -0,0 +1,31 @@
+
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x, 18.x]
```
It is not necessary to check version 14 in Ubnutu

suyuan32 commented:
Your workflow doesn't seem to be working.

suyuan32 commented:
> CC #106
> 
> Github CI/CD run error:
> 
> ```
> Error: Dependencies lock file is not found in XX 
> Supported file patterns: package-lock.json,npm-shrinkwrap.json,yarn.lock
> ```
> 
> It is necessary to discuss whether to add a `package-lock` file.

We use pnpm instead, it is pnpm-lock.yaml.

suyuan32 commented:
https://stackoverflow.com/questions/76664013/how-to-use-pnpm-instead-of-npm-in-github-actions

suyuan32 commented:
And you should push to dev branch

Jzow commented:
Link: 

1. https://github.com/vbenjs/vue-vben-admin/issues/3010
2. https://github.com/vitejs/vite/issues/14350

suyuan32 commented:
Too many commits, you should squash them into 1 or 2 commits. When you finish all the modification, leave a comment.

Jzow commented:
Fixed, this PR has done two things:

1. We have modified some `less` file naming conventions under the `design module` while also modifying the names referenced by` vue component` files. The purpose of this is that we should not follow the practice of non-standard `vben-ui` naming. We can make appropriate adjustments to facilitate future extensions of new `css` and `less` files that also need to comply with the regulations.
2. Add CI/CD for Github Action.

suyuan32 reviewed (commented):
ok
